src/global/types.h

```diff
@@ -993,6 +993,7 @@ typedef enum GAMEFLOW_OPTION {
     GF_START_SAVED_GAME = 7 << 6,
     GF_RESTART_GAME = 8 << 6,
     GF_SELECT_GAME = 9 << 6,
+    GF_START_GYM = 10 << 6,
```

Comment:
That's not what I meant. This enum is not necessary; I meant to discern between **new game** and **continue to next level**. (Both of which are out of scope for this pull request)

Reply:
On another thought let's keep things this way, they're pretty clean.